tasks/main.yml

```diff
@@ -16,3 +16,10 @@
     insertbefore: "^exit"
     state: present
   with_items: "{{ raspberry_pi_rc_local_options }}"
+
+- name: Configure options in /boot/cmdline.txt
+  replace:
+    path: /boot/cmdline.txt
```

Comment:
This is /boot/firmware/cmdline.txt on Ubuntu; would be great if that was an option!